docs/source/topics/frontier-settings.rst

```diff
@@ -202,6 +202,15 @@ Default: ``crawlfrontier.utils.fingerprint.sha1``
 
 The function used to calculate the ``domain`` fingerprint.
 
+.. setting:: OVERUSED_SLOT_FACTOR
+
+OVERUSED_SLOT_FACTOR
+------------------------
+
+Default: ``1.0``
+
+This is an amount of overall active connections to concurrent connections allowed in slot(per-domain/ip) after which slot
+ is considered overused.
```

Comment:
Can len(active) be greater than concurrency? I think it makes sense to note what the default argument means. If I'm not mistaken, now it means "never consider slots overused" because the inequality is strict, and it seems a number of active requests can't exceed the concurrency number. But maybe it can ;) and in this case the meaning of the default value is different.